rest_framework_tracking/mixins.py

```diff
@@ -40,21 +40,17 @@ def initial(self, request, *args, **kwargs):
         else:
             view_method = method.lower()
 
-        # save to log
-        try:
-            self.request.log = APIRequestLog.objects.create(
-                requested_at=now(),
-                path=request.path,
-                view=view_name,
-                view_method=view_method,
-                remote_addr=ipaddr,
-                host=request.get_host(),
-                method=request.method,
-                query_params=_clean_data(request.query_params.dict()),
-            )
-        except Exception:
-            # tracking db constraints should not create a failure on the API
-            return
+        # create log
+        self.request.log = APIRequestLog(
```

Comment:
As far as I know, this change would only create the object in memory and not persist to the db.